doc/design/validation.md

```diff
@@ -0,0 +1,25 @@
+# Requirements for being a valid DANDI asset
+
+## General
+1. All assets must have a non-zero file size.
+
+## NWB files
+- subject id
+- age in iso8601 interval format or datetime of birth
+- age reference 
+- sex
+
+### NWB organization disambiguation requirements
+
+At least one of the following should be present. in case of having to disambiguate
+- session
+- sample
+- cell
```

Comment:
we don't have a notion of cell in core either

Reply:
is it in `ndx-dandi-icephys` or some other extension?